### PR TITLE
Update to UG and DG

### PR DESCRIPTION
**Pull request by ehz0ah**
UG:
- Added FAQs
- Added Accepted File Formats Section
- Clean up Quick Start Guide

DG:
- Added more NFRs
- Added instructions for manual testing
- Update target user profile and value proposition

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -448,13 +448,23 @@ The program continues to run afterward, prompting the user for commands.
...
+
+- **Efficiency**: Automated workflows reduce the time and effort required to match students with available SEP locations, freeing up administrators to focus on more strategic tasks.
+- **Accuracy**: Data-driven decision-making ensures that allocations are based on objective criteria, minimizing errors and bias.
+- **Convenience**: The user-friendly interface and clear documentation make it easy for administrators to navigate the system and carry out their tasks effectively.
+- **Scalability**: The application can handle a large number of student applications and SEP locations, making it suitable for both small and large cohorts.
+- **Transparency**: Detailed tracking and reporting features provide visibility into the allocation process, allowing for better oversight and accountability.
+- **Support**: Comprehensive support and maintenance services ensure that the application remains reliable and up-to-date, adapting to any changes in the SEP process or requirements.
+
+By integrating these features, the application significantly enhances the convenience and effectiveness of managing SEP allocations, ultimately benefiting both administrators and students.
```
Comment (isaacsaw25) [inline docs/DeveloperGuide.md:467]: Excellent explanation!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -20,26 +20,30 @@ FindMySEP is a Command Line Interface (CLI) tool designed for admins handling th
   - [Revert Command](#revert-allocation-outcome-revert)
   - [Exit Command](#exit-program-bye-exit-quit)
 - [FAQ](#faq)  
-- [Command Summary](#command-summary)  
+- [Command Summary](#command-summary)
+- [Accepted File Format]()
```
Comment (isaacsaw25) [inline docs/UserGuide.md:24]: Is there a reason the link is blank?

File: src/main/java/ui/Messages.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package ui;
 
 public enum Messages {
-    WELCOME("Hi! Welcome to FindOurSEP! Enter 'help' for the list of commands."),
```
Comment (isaacsaw25) [inline src/main/java/ui/Messages.java:4]: I feel that leaving the help portion might be helpful for users.

File: docs/UserGuide.md (hunk 2 above)
Comment (ehz0ah) [reply docs/UserGuide.md:24]: LOL nice catch

File: src/main/java/ui/Messages.java (hunk 3 above)
Comment (isaacsaw25) [reply src/main/java/ui/Messages.java:4]: 알갰어 👍 기다리고 있어 👀

**Review by isaacsaw25 — APPROVED** [review]
다 완벽하다! 잘 했어! 💪 (Well done it's perfect)

**Review by isaacsaw25 — APPROVED** [review]
LGTM

**Pull request merged**